styles/npm/catppuccin.user.css

```diff
@@ -1035,7 +1036,7 @@
     }
 
     /* Input elements  */
-    #signup_email_error_message {
+    #signup_email_error_message, ._3714e37c {
```

Comment:
i'll use ``#signup_password_error_message``

Reply:
Why would you not use that in the first place?? It's human readable, not likely to change - unlike the autogenerated classname of `_3714e37c` - and already matches/goes with the other ID there (`signup_email_error_message`)?

Reply:
This principle of picking better selectors **goes for every userstyle we have**. Please be really careful and specific because this is how we make userstyles maintainable and working in the long term!

Reply:
> Why would you not use that in the first place?? It's human readable, not likely to change - unlike the autogenerated classname of `_3714e37c` - and already matches/goes with the other ID there (`signup_email_error_message`)?

i didn't see the id when i was inspecting the signup page